packages/webkit2gtk_5.rb

```diff
@@ -1,101 +1,180 @@
-require 'package'
-
 class Webkit2gtk_5 < Package
   description 'Web content engine for GTK'
   homepage 'https://webkitgtk.org'
-  @_ver = '2.32.4'
-  version @_ver
+  case ARCH
+  when 'x86_64'
+    version '2.38.1'
+  when 'i686'
+    version '2.38.0'
+  when 'aarch64', 'armv7l'
+    version '2.32.4'
+  end
```

Comment:
This is not preferable.

Reply:
What would you suggest? Just taking out the binaries section for the other architectures?

Reply:
This was changed.